src/components/common/request-details.vue

```diff
@@ -3,59 +3,77 @@
     <section>
       <h4>Requested Name <span v-if="nameChoices.name2">Choices</span></h4>
       <ul class="pl-0">
-        <li v-if="!nameChoices.name2">{{nameChoices.name1}} {{nameChoices.designation1}}</li>
+        <li v-if="!nameChoices.name2">
+          {{ nameChoices.name1 }} {{ nameChoices.designation1 }}
+        </li>
         <template v-else>
-          <li v-if="nameChoices.name1">1. {{nameChoices.name1}} {{nameChoices.designation1}}</li>
-          <li v-if="nameChoices.name2">2. {{nameChoices.name2}} {{nameChoices.designation2}}</li>
-          <li v-if="nameChoices.name3">3. {{nameChoices.name3}} {{nameChoices.designation3}}</li>
+          <li v-if="nameChoices.name1">
+            1. {{ nameChoices.name1 }} {{ nameChoices.designation1 }}
+          </li>
+          <li v-if="nameChoices.name2">
+            2. {{ nameChoices.name2 }} {{ nameChoices.designation2 }}
+          </li>
+          <li v-if="nameChoices.name3">
+            3. {{ nameChoices.name3 }} {{ nameChoices.designation3 }}
+          </li>
         </template>
       </ul>
     </section>
 
     <v-row>
-      <v-col col="6">
+      <v-col class="col">
         <h4>Applicant Information</h4>
         <ul class="pl-0">
           <!-- If there's no contact person (agent / lawyer / etc.) the applicant is the contact -->
-          <li>{{`${applicantName}`}}</li>
+          <li>{{ `${applicantName}` }}</li>
           <li>
-            {{`${applicant.addrLine1 ? applicant.addrLine1 : ''} ${applicant.addrLine2 ? applicant.addrLine2 : ''}`}}
+            {{ `${applicant.addrLine1 ? applicant.addrLine1 : ''} ${applicant.addrLine2 ? applicant.addrLine2 : ''}` }}
           </li>
           <li>
-            {{`${applicant.city ? applicant.city : ''}, ${applicant.stateProvinceCd ? applicant.stateProvinceCd : ''}`}}
+            {{ `${applicant.city ? applicant.city : ''}, ${applicant.stateProvinceCd ? applicant.stateProvinceCd : ''}` }}
           </li>
           <li>
-            {{`${applicant.countryTypeCd === Location.CA ? 'Canada' : applicant.countryTypeCd},
-             ${applicant.postalCd ? applicant.postalCd : ''}`}}
+            {{ `${applicant.countryTypeCd === Location.CA ? 'Canada' : applicant.countryTypeCd},
+             ${applicant.postalCd ? applicant.postalCd : ''}` }}
           </li>
         </ul>
       </v-col>
 
-      <v-col col="6">
+      <v-col class="col">
```

Comment:
I will test this before merging.

Reply:
Same as above. (Works fine.)

Note that `class="col"` is NOT the final code. This is an old snapshot.